### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by supernovahs**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -22,6 +22,7 @@ use ark_ec::{
     pairing::{Pairing, PairingOutput},
     AffineRepr, CurveGroup,
 };
+use ark_ec::AdditiveGroup;
```
Comment (anupsv) [inline src/lib.rs]: This doesn't seem required.

File: Cargo.toml [hunk 2]
```diff
@@ -12,32 +12,32 @@ exclude = [
 ]
 
 [dependencies]
```
Comment (anupsv) [inline Cargo.toml:15]: would be good to add `rust-version = "1.75"`

Comment (supernovahs) [reply Cargo.toml:15]: can i do 1.79 ? since sdk uses it

File: src/lib.rs (hunk 1 above)
Comment (supernovahs) [reply src/lib.rs]: it was needed in tests. moved it there

File: src/lib.rs [hunk 3]
```diff
@@ -218,6 +218,7 @@ pub fn aggregate_public_keys(public_keys: &[G2Affine]) -> Result<G2Affine, BLSEr
 #[cfg(test)]
 mod test {
     use super::*;
+use ark_ec::AdditiveGroup;
```
Comment (anupsv) [inline src/lib.rs:221]: Looks like this got added back ?

Comment (supernovahs) [reply src/lib.rs:221]: compiler throws error if i remove it. 
```
help: trait `AdditiveGroup` which provides `double` is implemented but not in scope; perhaps you want to import it
    |
220 +     use ark_ec::AdditiveGroup;
    |

error[E0599]: no method named `double` found for struct `QuadExtField` in the current scope
   --> src/lib.rs:325:53
    |
325 |         invalid_public_key.y = invalid_public_key.y.double(); // This makes it invalid
    |                                                     ^^^^^^ method not found in `QuadExtField<Fp2ConfigWrapper<Fq2Config>>`
    |
```

Comment (supernovahs) [reply src/lib.rs:221]: @anupsv

Comment (anupsv) [reply src/lib.rs:221]: Interesting. ok.

**Review by anupsv — APPROVED** [review]

**Pull request merged**